addons/mass_mailing/static/src/js/mailing_portal.js

```diff
@@ -1,189 +1,192 @@
 /** @odoo-module **/
 
-    import session from "web.session";
-    import ajax from "web.ajax";
-    import core from "web.core";
-    import "web.dom_ready";
-    import { sprintf, escape } from "@web/core/utils/strings";
+import session from "web.session";
+import ajax from "web.ajax";
+import core from "web.core";
+import { sprintf, escape } from "@web/core/utils/strings";
+import { whenReady } from "@odoo/owl";
 
-    var _t = core._t;
+var _t = core._t;
 
+whenReady(() => {
     var email = $("input[name='email']").val();
     var mailing_id = parseInt($("input[name='mailing_id']").val());
     var res_id = parseInt($("input[name='res_id']").val());
     var token = (location.search.split('token' + '=')[1] || '').split('&')[0];
 
-if ($('.o_unsubscribe_form').length) {
```

Comment:
@mcm-odoo could you reapply the previous indentation change here? it was badly indented, but because of that, there is a security check that fails, and I don't think it's worth it

Reply:
security check has already been removed by xmo :smile: